var/spack/repos/builtin/packages/coin3d/package.py

```diff
@@ -43,6 +50,26 @@ class Coin3d(AutotoolsPackage):
     variant("debug", default=False, description="Make debug build")
     variant("symbols", default=False, description="Enable debug symbols")
 
+    def url_for_version(self, version):
+        if version >= Version("4.0.0"):
+            url = "https://github.com/coin3d/coin/releases/download/Coin-{0}/coin-{0}-src.tar.gz"
+        else:
+            url = "https://github.com/coin3d/coin/archive/Coin-{0}.tar.gz"
+        return url.format(version.dotted)
+
+
+class CMakeBuilder(spack.build_systems.cmake.CMakeBuilder):
+    def cmake_args(self):
+        args = [
+            self.define_from_variant("COIN_BUILD_DOCUMENTATION_MAN", "man"),
+            self.define_from_variant("COIN_BUILD_DOCUMENTATION_CHM", "html"),
+            self.define_from_variant("COIN_BUILD_MAC_FRAMEWORK", "framework"),
+            self.define_from_variant("COIN_BUILD_SHARED_LIBS", "shared"),
```

Comment:
No more symbols/debug? Should we make those variants conditional on version? I'm guessing these are controlled by build_type.

Reply:
correct